livecheck/settings.py

```diff
@@ -46,6 +45,18 @@ class LivecheckSettings:
     regex_version: dict[str, tuple[str, str]]
     restrict_version: dict[str, str]
     sync_version: dict[str, str]
+    '''Settings from command line flag.'''
+    auto_update_flag: bool = False
+    debug_flag: bool = False
+    development_flag: bool = False
+    git_flag: bool = False
+    keep_old_flag: bool = False
+    progress_flag: bool = False
+    '''Setting form internal process.'''
```

Comment:
```suggestion
    # Internal settings.
```

These should just be normal `#` comments.

What you are doing here when you have it as `'''` here is making it into documentation for the `progress_flag` above.